src/grt/src/fastroute/include/FastRoute.h

```diff
@@ -117,7 +117,7 @@ class FastRouteCore
                 int max_layer,
                 float slack,
                 std::vector<int>* edge_cost_per_layer);
-  void removeNet(odb::dbNet* db_net);
+  void removeNet(odb::dbNet* db_net, bool release_resources);
```

Comment:
It might be better to have a mergeNet here rather than remove but keep the resources which feels semantically odd.